src/middlewared/middlewared/plugins/audit/utils.py

```diff
@@ -195,3 +197,17 @@ def requires_python_filtering(
 
 
 AUDIT_TABLES = {svc[0]: generate_audit_table(*svc) for svc in AUDITED_SERVICES}
+
+
+async def setup_truenas_verify(sysver: str) -> tuple:
```

Comment:
Since truenas_verify is installed as a module, you can "import truenas_verify" and call "truenas_verify.main()"